src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -35,6 +36,8 @@
  */
 final class EagerLoadingExtension implements QueryCollectionExtensionInterface, QueryItemExtensionInterface
 {
+    use ShouldEagerLoad;
```

Comment:
:stuck_out_tongue: I never liked traits, if you have another solution/name in mind please tell me I'll do another pull request ;)